go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/TankerHQ/identity-go/v3
 
-go 1.17
+go 1.18
```

Comment:
Could you please also run `go mod tidy` and commit changes on `go.sum`?

Additional nitpick: this does not seem to be a deprecation but rather a drop of a previously supported version. Maybe rename the commit to `deps(go): drop support for go 1.17`?